Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -28,6 +28,7 @@ def handle_error(message, is_return_error):
         return_error(message)
     else:
         demisto.results(message)
+        sys.exit(0)
```

Comment:
I think it's fine, @lperryp ?

Reply:
@erezh31 I think so too (if it won't return an error )